src/main/java/dan200/computercraft/api/turtle/ITurtleAccess.java

```diff
@@ -122,19 +122,37 @@
      * @param dyeColour 0-15 to dye the turtle one of the 16 standard Minecraft <em>dye</em> colours, or -1 to remove
      *                  the dye from the turtle.
      * @see #getDyeColour()
+     * @see #setColour(int)
      */
     void setDyeColour( int dyeColour );
```

Comment:
I don't mind just removing these two methods. I'm not aware of any mods that used them and small API changes are expected between minecraft versions.